week1-IntroAndSetupGuide.md

```diff
@@ -0,0 +1,49 @@
+  1.2. Check the [git cheatsheet](git-cheatsheet.md#create-a-branch) for instructions.
+ 2. Install the latest Long Term Support (LTS) version of node js from https://nodejs.org/en/. Currently this is version 10.  
+  2.1. Avoid non LTS versions, e.g. version 11, since they tend to have bugs.  
+
+## Starting the app 
+ This section will give you the steps to generate the base code to start our application. 
+ We'll start by creating a [React](https://reactjs.org/) app: 
+ 
+1. Launch Visual Studio Code and use `File -> Open Folder` and open the folder `front-end`. It will be under the folder we created on the previous sections (Ex: CodessInTheClassroom).  
+2. Open a terminal by clicking on `Terminal -> New Terminal`. 
+3. Create and start a basic typescript based web application. Run the following commands:
+	```
+	 > npx create-react-app web-application --typescript
+	 > cd web-application
+	 > npm start
+	```
+	 See [create-react-app on github](https://facebook.github.io/create-react-app/docs/getting-started)  for more info.
+4. Navigate to http://localhost:3000/ to see the running application.
+
+### Commit and create a PullRequest
+
+So far, we've generated a basic react app, which we will build on, to create the Pet Adoption Service front-end. If you look at your source code now, you'll see a lot of files that were generated by the tools and which are simply boilerplate code: stuff we usually don't change, but yet necessary for the app to run (kind of like the nuts and bolts of our system).
+We will commit and push this code to our master branch now for two reasons: 1. This is a checkpoint for us, meaning we have stable code which starts up a basic react app and 2. From now on, we will start customizing this code to our needs, so we want to have a clear distinction from the code that we generated from the third party tools and the code we'll create as part of the requirements of our system.
+
+1. Commit your changes by following [these instructions](git-cheatsheet.md#commit).
+3. With all the changes committed, the next step is to Push the code: check the steps [on the cheatsheet](git-cheatsheet.md#push).
+4. Now that the code is available at GitHub, let's create a [Pull Request](https://help.github.com/en/articles/about-pull-requests) by following the instructions [here](https://help.github.com/en/articles/creating-a-pull-request#creating-the-pull-request).  
+  4.1 Add a few people from the room as your reviewers: At this stage, we will only check if the files in the Pull Request you're reviewing are the same as your files. If all the steps were followed correctly, everyone should have the same generated code, therefore, there's no need for a deeper review.  
+	4.2 Once you get at least two approvals, [merge](https://help.github.com/en/articles/merging-a-pull-request#merging-a-pull-request-on-github) the PullRequest. Select the `Merge pull request`, from the drop-down list (item 3 of the referred documentation).
```

Comment:
Tiny thing, but from a logistics perspective I think we should expect that people are going to have one other person review their code rather than two.